pkg/api/store/hpa/store.go

```diff
@@ -0,0 +1,56 @@
+package hpa
+
+import (
+	"time"
+
+	"github.com/rancher/norman/store/transform"
+	"github.com/rancher/norman/types"
+	"github.com/rancher/norman/types/values"
+)
+
+const (
+	condition = "ScalingActive"
+	status    = "False"
+	reason    = "FailedGetResourceMetric"
+)
+
+func NewIgnoreTransitioningErrorStore(store types.Store, d time.Duration, expectedState string) types.Store {
+	return &transform.Store{
+		Store: store,
+		Transformer: func(apiContext *types.APIContext, schema *types.Schema, data map[string]interface{}, opt *types.QueryOptions) (map[string]interface{}, error) {
+			created, ok := values.GetValue(data, "created")
+			if !ok {
+				return data, nil
+			}
+			t, err := time.Parse(time.RFC3339, created.(string))
```

Comment:
Can you put a check for this to see if transitioning == "error" before parsing the date? You could even move parsing the date to below the point where you find the offending condition so that you only have to parse the date if you know you are in this scenario?

I don't know how expensive time.Parse is, but seems like it would be good to avoid doing it if we are not in a transitioning == error state